armory/datasets/package.py

```diff
@@ -18,16 +18,30 @@ def package(
     """
     Package a built dataset as .tar.gz, return path
     """
-    version, data_dir, built_data_dir, subdir = build.build_info(
+    version, data_dir, built_data_dir, subdir, builder_configs = build.build_info(
         name, version=version, data_dir=data_dir
     )
 
     data_dir = Path(data_dir)
-    expected_dir = data_dir / name / version
-    if not expected_dir.is_dir():
-        raise FileNotFoundError(f"Dataset {name} not found at {expected_dir}")
-    tar_full_filepath = common.get_cache_dataset_path(name, version)
 
+    if not builder_configs:
+        tar_list = [str(Path(name) / version)]
+    else:
+        # including .config, which contains metadata.json that tfds refers to for default_config_name of a given dataset
+        # .config and metadata.json are hardcoded values for tfds v4.6.0
+        # should subdirectories exist in a dataset, user has the option to choose a subdirectory
```

Comment:
I think the comments starting at # should subdirectories ... can be removed. The context added in the first two lines is helpful and sufficient, and we should have docs elsewhere (datasets.md, configuration_files.md, and/or load() docstring) explaining what the `config` input is

Reply:
resolved with b6cd933f